tests/frontend/test_collections.py

```diff
@@ -495,21 +497,14 @@ def test_create_collection_with_empty_custom_url(selenium, base_url):
 @pytest.mark.serial
 @pytest.mark.nondestructive
 def test_create_collection_with_already_used_url(selenium, base_url, variables):
+    """Try to create a collection with a name and URL that are already in use
+    by another of the same user's collections"""
     collections = Collections(selenium, base_url).open().wait_for_page_to_load()
     collections.login('collection_user')
     collections.click_create_collection()
-    collections.create.set_name('abc')
-    collections.create.slug_label_element.clear()
-    collections.create.set_slug('abc')
+    collections.create.set_name('reused-slug')
     collections.create.save_collection()
     collections.collection_detail.wait_for_details_to_load()
-    # go back and try to create another collection with different name, but same URL
-    collections.driver.get('https://addons.allizom.org/en-US/firefox/collections/')
-    collections.click_create_collection()
-    collections.create.set_name('def')
-    collections.create.slug_label_element.clear()
-    collections.create.set_slug('abc')
-    collections.create.save_collection()
     assert variables['collection_reused_url_warning'] in collections.create.warning_text
```

Comment:
This test was having too many unnecessary steps. I've made a more simple version for it which achieves the same goal.